pixels.go

```diff
@@ -0,0 +1,98 @@
+package waifu2x
+
+import (
+	"fmt"
+	"math"
+)
+
+type Pixels struct {
```

Comment:
__[gometalinter]__
* [golint] exported type Pixels should have comment or be unexported